managing_providers/_topics/embedded_workflows.md

```diff
@@ -165,11 +165,11 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
 
    The workflow code must be in the Amazon States Language (ASL) format and follow its supported specifications. For more information about Amazon States Language and its specification, see [Amazon States Language Guide](https://states-language.net/).
 
-   **Note**: The current implementation of the Amazon States Language does not support certain **Map** states. See [Unsupported States Language Features](#unsupported-states-language-features).
+   **Note**: The current implementation of the Amazon States Language does not support certain **Map** states. For more information, see [Unsupported States Language features](#unsupported-states-language-features).
 
-   #### Unsupported States Language Features
+   #### Unsupported Amazon States Language features
 
-    The following are not yet supported for Floe:
+    The following features are not supported for Amazon States Language (Floe):
```

Comment:
Maybe:
```suggestion
    The following Amazon States Language features are not supported for by Floe:
```

or 
```suggestion
    The following features of Amazon States Language are not supported by Floe:
```

Reply:
People won't know what Floe is since this is the first reference....
I like the second suggestion otherwise.